04.class/MemoApp.js

```diff
@@ -0,0 +1,35 @@
+import pkg from "enquirer";
+const { prompt } = pkg;
+
+export default class MemoApp {
+  constructor(memos) {
+    this.memos = memos;
+  }
+
+  async choose() {
+    //enquireを使ってメモタイトルを表示させるために、
+    //convert()で{ name: 'memoTitle', value: 'memoID' }に変換する
+    const choices = this.memos.map((memo) => memo.convert());
+    const lines = [
+      {
+        type: "select",
+        name: "memo",
+        message: "Choose a memo you want to see:",
+        choices: choices,
```

Comment:
プロパティの名前と値が同じ時は省略記法を使いましょう。

```suggestion
        choices,
```

Reply:
[7692742](https://github.com/Judeeeee/js-practices/pull/4/commits/76927429cd643b478243c1728082795aa144863e)で修正しました。